src/tooltip/tooltip.js

```diff
@@ -16,4 +16,6 @@ const toggleTooltip = (event) => {
   }
 };
 
-export default toggleTooltip;
+const enableTooltips = () => document.addEventListener('mousemove', toggleTooltip);
```

Comment:
It's better rename var because U can use futher enableTooltip.remove(): to clean redundant listeners